store/mock.go

```diff
@@ -104,7 +104,8 @@ func (m *MockStore) HasAnyBlock() bool {
 
 func (m *MockStore) IterateAccounts(consumer func(*account.Account) (stop bool)) {
 	for _, a := range m.Accounts {
-		stopped := consumer(&a)
+		acc := a
+		stopped := consumer(&acc)
 		if stopped {
 			return
 		}
```

Comment:
i don't see why do we need a extra redundant variable in thread safe functionality

at line 117 as well

Reply:
It's security linting error. We pass the object with its pointer to outside and it might be changed.